src/Hangman/Domain/Events/NewGameStarted.event.ts

```diff
@@ -0,0 +1,6 @@
+import { IEvent } from '@nestjs/cqrs';
+import { GameDto } from '../AggregateRoot/GameDto';
+
+export class NewGameStartedEvent implements IEvent {
+  constructor(public readonly gameDto: GameDto) {}
```

Comment:
1: Het veel duidelijker als je het schrijft
2: In events kan je geen value object opslaan want (en dit is heel theoretisch). Stel dit is een Java of PHP applicatie en je gaat die object serializen dan krijg je iets wat voor die taal specifiek is en daardoor zijn die event niet meer zo maar te gebruiken door applicatie geschreven in python.
Er zijn truukjes om dit probleem dit te omzeilen door zelf een serializer te maken die value objects omzet naar JSON en terug.
Dit is een beetje theoretisch allemaal

Reply:
```
constructor(gameId, playerId, wordToGuess, maxGuesses) {
 // en eigenlijk moet je hier ook weer opnieuw valideren
}
```